contributing.md

```diff
@@ -0,0 +1,54 @@
+
+If you are out of ideas,still you can contribute.Reporing Bugs, Documentation , Suggesting Improvements etc. are few of contributions for you except PRs.
+
+
+### Reporting New Issues - Bugs
+
+This section guides how to make a Bug report which can be understand by the community and enables to reproduce the behaviour.
+
+GitHub tracked Bugs as Issues.Firstly follow the [ GitHub Issues ](https://guides.github.com/features/issues) to get a overrall idea on GitHub Issues.After you understood what is going on,then create a new Issue on the particular repository and provide necessary Information.
+
+#### Please follow below instructions to submit a good Bug Report
+
+* ** Always Use a Clear and Descriptive Title ** - This helps contributors to identify the problem quickly
+* ** Describe the steps which leads to the certain Issue ** - Better if describe the steps in brief.
+* ** Provide Screenshots and if can GIFs  ** - Though is an additional , it makes the contributors to understand the problem clearly.
+* ** Describe the behavior you observed after following the steps ** - Mention the exact problem corresponding to the step.
+* ** Explain which behaviour you are expected to see and Why so ? ** - This guides what to fix with the issue.
+
+
+### Documentation
+
+This section guides how to contribute projects by Documentation. Documentation consists of Contributing.md,Code_Of_Conduct.md etc.Follow below instructions before create Documents for the project.
+
+* ** Follow [GitHub CODE_OF_CONDUCT]( https://help.github.com/articles/adding-a-code-of-conduct-to-your-project ) ** before contributing.
+* ** Make sure to select correct Licence before adding ** - If have any problems don't hesitate to contact via mail lists of organization.
+Follow [Licensing a repository](https://help.github.com/articles/licensing-a-repository/) for a better understanding.
+
+[Template of Good_Contributing.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) check for further understanding.
+
+
+### Suggesting Improvements
+
+Improvements also same as GitHub Issues.
+
+GitHub tracked Improvements as Issues.Firstly follow the [ GitHub Issues ](https://guides.github.com/features/issues) to get a overrall idea on GitHub Issues.After you understood what is going on,then create a new Issue on the particular repository and provide necessary Information.
+
+* ** Always Use a Clear and Descriptive Title ** - This helps contributors to identify the Improvement quickly
+* ** Describe the steps which leads to the certain Improvements ** - Better if describe the steps in brief.
+* ** Explain current behaviour if the minor version already implemented and Explain what you expect instead ** 
+
+
+#### Lets begin to Contribute
+
+Firstly Fork the repository to your GitHub account.Then Clone it into local PC.Now you are ready to contribute...Cheers!!!

```

Comment:
Needs more information here. like a simple description about fork model. How to branch out. when to send PR etc.